GLES2 Shader Compile Fail due to undefined variable "m_LIGHT"

**Godot version:**
3.1 Beta 2


**OS/device including version:**
GTX 960m
Windows 10

**Issue description:**
Shader works on GLES3, but does not work on GLES2. Not doing anything particularly crazy

 SceneShaderGLES2: Fragment shader compilation failed:
0(552) : error C1503: undefined variable "m_LIGHT"


**Steps to reproduce:**
I've attached a reproduction project w/ shader already set up.

**Minimal reproduction project:**
[GLES2_SHADERFAIL.zip](https://github.com/godotengine/godot/files/2803780/GLES2_SHADERFAIL.zip)

Confirmed in the current master branch (ac01aef). Relevant portion of the shader code:
```
[...]
#if defined(USE_LIGHT_SHADER_CODE)
544:    // light is written by the light shader
545: 
546:    vec3 normal = N;
547:    vec3 albedo = diffuse_color;
548:    vec3 light = L;
549:    vec3 view = V;
550: 
551:    /* clang-format off */
552: {
553:    float m_NdotL = dot(normal, m_LIGHT);
554:    float m_diff = ((m_NdotL * 0.5) + 0.5);
555:    float m_ts = dot(normal, view);
556:    m_ts = clamp(mix((1.0 - step(m_ts, m_amount)), m_ts, 0.5), 0.0, 1.1);
557:    vec3 m_mx = mix(vec3(0.0,0.0,0.0), vec3(1.0,1.0,1.0), m_ts);
558:    diffuse_light += (m_mx * (((albedo * (1.0 - (roughness * 0.8))) * light_color) * (attenuation * 2.0)));
559:    specular_light += (m_mx * (((albedo * (1.0 - (roughness * 0.8))) * light_color) * (attenuation * 2.0)));
560: }
561: 
562: 
563:    /* clang-format on */
564: 
565: #else
[...
0(553) : error C1503: undefined variable "m_LIGHT"

```

--- a/drivers/gles2/shader_compiler_gles2.cpp
+++ b/drivers/gles2/shader_compiler_gles2.cpp
@@ -869,6 +869,7 @@ ShaderCompilerGLES2::ShaderCompilerGLES2() {
 	//for light
 	actions[VS::SHADER_SPATIAL].renames["VIEW"] = "view";
 	actions[VS::SHADER_SPATIAL].renames["LIGHT_COLOR"] = "light_color";
+	actions[VS::SHADER_SPATIAL].renames["LIGHT"] = "light";
 	actions[VS::SHADER_SPATIAL].renames["ATTENUATION"] = "attenuation";
 	actions[VS::SHADER_SPATIAL].renames["DIFFUSE_LIGHT"] = "diffuse_light";
 	actions[VS::SHADER_SPATIAL].renames["SPECULAR_LIGHT"] = "specular_light";

